packages/edge-gateway/src/durable-objects/gateway-metrics.js

```diff
@@ -85,6 +86,16 @@ export class GatewayMetrics0 {
       this.gatewayMetrics.totalWinnerRequests += 1
     }
 
+    // Only update response time histogram if response is successful
+    if (stats.status === SUCCESS_STATUS) {
+      this._updateResponseTimeHistogram(stats)
```

Comment:
It's now not obvious that this statistic is for successful responses - perhaps we should rename to reflect the change here?